botorch/optim/optimize.py

```diff
@@ -152,6 +153,11 @@ def optimize_acqf(
         return X, acq_value
 
     if batch_initial_conditions is None:
+        if raw_samples is None:
+            raise ValueError(
```

Comment:
this isn't covered by the tests, can you please add a `with self.assertRaises` check for this?